torchtext/datasets/text_classification.py

```diff
@@ -44,19 +45,17 @@ def _csv_iterator(data_path, ngrams, yield_cls=False):
 def _create_data_from_iterator(vocab, iterator, include_unk):
     data = []
     labels = []
-    with tqdm(unit_scale=0, unit='lines') as t:
-        for cls, tokens in iterator:
-            if include_unk:
-                tokens = torch.tensor([vocab[token] for token in tokens])
-            else:
-                token_ids = list(filter(lambda x: x is not Vocab.UNK, [vocab[token]
-                                        for token in tokens]))
-                tokens = torch.tensor(token_ids)
-            if len(tokens) == 0:
-                logging.info('Row contains no tokens.')
-            data.append((cls, tokens))
-            labels.append(cls)
-            t.update(1)
+    for cls, tokens in iterator:
+        if include_unk:
+            tokens = torch.tensor([vocab[token] for token in tokens])
+        else:
+            token_ids = list(filter(lambda x: x is not Vocab.UNK, [vocab[token]
+                                    for token in tokens]))
+            tokens = torch.tensor(token_ids)
+        if len(tokens) == 0:
+            logging.info('Row contains no tokens.')
+        data.append((cls, tokens))
+        labels.append(cls)
```

Comment:
Just mean to confirm that there was a decision to materialize the the list, right?

Reply:
We can switch this over to the generator style. Probably a worthwhile optimizing and more consistent. @zhangguanheng66 - can you look into this?

Reply:
Yes. I could fix it. Users won't see the change.

Reply:
Fixed